addons/website/views/snippets/s_company_team.xml

```diff
@@ -20,35 +23,38 @@
                         </div>
                     </div>
                 </div>
-                <div class="col-lg-6 pt24 pb24">
+                <div class="col-lg-6 pt32 pb32">
```

Comment:
Let's give this guy a real `name` ;)

```suggestion
                <div class="col-lg-6 pt32 pb32" data-name="Member">
```
| before | after |
|--------|--------|
| <img width="286" alt="image" src="https://github.com/user-attachments/assets/e9d3c3ec-3d4b-4646-aabd-b70806072838"> | <img width="283" alt="image" src="https://github.com/user-attachments/assets/3a0ffb66-671d-499a-a7cb-04fa24a064c7"> |